RELEASE_PROCESS.md

```diff
@@ -39,15 +41,34 @@ Every commit to chef-server master is tested against a full pedant
 run. However, upgrade testing must still be done in advance of the
 release:
 
-- [ ] Run the Chef Delivery-based automated upgrade testing
-  * Checkout the `chef-server-acceptance` project from delivery.chef.co
-  * Update the `chef_server_test_url-override` attributes in
-    `.delivery/build/attributes/default.rb` to the URL corresponding to
-    the latest package you'd like to test at artifactory.chef.co
-  * Commit the change and run `delivery review`
+- [ ] Run the Chef Delivery-based automated upgrade testing.
+  * Checkout the `chef-server-acceptance` project from delivery.chef.co.
+  * Update relevant attributes in `.delivery/build/attributes/default.rb`:
+    * Update `chef_server_test_url-override` to be the URL corresponding to
+      the latest package you'd like to test at artifactory.chef.co (probably
+      a git poll build that you want to ship as the next version).
+    * Update `chef_server_latest_released_version` to match the last release of
+      the Chef Server. Should match the version at the [download page](https://downloads.chef.io/chef-server/).
+    * If delivery.chef.co builders are too constrainted to run the full matrix of 10 jobs
```

Comment:
I'd put this "if" block below/next to the "If you had no failures" block.  Because I wouldn't know what "run a build" was until I read the instructions below and then realized that "running a build" was the same as doing the delivery review/wait for lint/run approve/wait for the long running acceptance tests.